hub/src/server/authentication.js

```diff
@@ -81,9 +83,15 @@ export class V1Authentication {
       validateScopes(scopes)
     }
 
+    let payloadCreationDate = (Date.now()/1000|0)
```

Comment:
Nitpick: could just be `const paloadCreationDate = creationDate || (Date.now()/1000|0)`